projects/novo-examples/src/updates/v10.md

```diff
@@ -0,0 +1,48 @@
+---
+section: updates
+page: v10-announce
+title: v10
+order: 5
+tag: new
+---
+
+🎉 April 2024 (version 10)
+===========================
+
+**Announcement**: Novo Elements is now on Angular 17!
+
+This brings Novo Elements up to the latest versions of both Angular and Typescript. We have also removed the long-since deprecated Table component and Dragula add-on. 
```

Comment:
This brings Novo Elements up to the latest versions of both Angular and Typescript. 
Some key highlights include:
- Removing the long-since deprecated Table component.
- Removing the Dragula add-on.